### ISSUE TITLE
Unable to open "Calendar" when Talkback is ON in Android

### ISSUE THREAD

**Issue by GayatriDeviP** [microsoft/fluentui#22301]
### Environment Information

- **Package version(s)**: Latest
- **Browser and OS versions**: Android 11

### Describe the issue:
Unable to open "Calendar" when Talkback is ON in Android

#### Actual behavior:
Not able to open calendar when talkback is ON in Android.

#### Expected behavior:
It should open the calendar when talkback is ON in Android.

### please provide a codepen repro:
https://codepen.io/GayatriDeviP/pen/MWrvbpW

Repro steps:
1. Open the codepen in Android and turn on talkback.
2. Navigate to the control rendered.
3. Now verify whether the calendar is opened or not

**Comment by gouttierre** [comment]
Thank you for filing this item with us. I'll add this to our contributors queue as it’s low on our priority list and we’re likely not going to get to it for a while as we’re currently focused on larger coherence work items and enhancements to our performance. 
In the meantime, if you want to contribute towards this request, please share your PR and our dev team will review it.
